README.md

```diff
@@ -5,9 +5,15 @@
 # Hue-sensors-HASS
 [FOR COMMUNITY SUPPORT PLEASE USE THIS THREAD](https://community.home-assistant.io/t/hue-motion-sensors-remotes-custom-component)
 
-Place the custom_components folder in your configuration directory (or add its contents to an existing custom_components folder). You need to set up [Hue bridge](https://www.home-assistant.io/components/hue/) first.
+## Installation
 
-Hue dimmer remotes can be used for a click and long press (hold button for 2 sec and see LED blink twice).
+Place the `custom_components` folder in your configuration directory (or add its contents to an existing `custom_components` folder). You need to set up your [Hue bridge](https://www.home-assistant.io/integrations/hue) first.
+
+Alternatively it can be installed via [HACS](https://hacs.xyz/).
+
+## Configuration
+
+Once installed you need to setup the `huesensor` integration in your configuration.
```

Comment:
@robmarkcole I think we should also explain why each entry is configured.

Reply:
what do you suggest we add?

Reply:
Perhaps explaining why adding each one... e.g what it gives. Perhaps It's not required though :+1:

Reply:
can open that as a second PR if you think it has merit :-)